.github/workflows/tests.yml

```
@@ -191,33 +182,32 @@ jobs:
             -   name: Checkout code
                 uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd # v6
                 with:
-                    # Prefer real LFS payloads for PubChem VCR smoke. If the account
-                    # LFS budget is exhausted, continue and skip the VCR-bound smoke.
+                    # Prefer real LFS payloads for PubChem VCR smoke. Fail closed
+                    # when required cassette payloads cannot be materialized (#7493).
                     lfs: false
 
             -   name: Fetch Git LFS objects for VCR cassettes
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)